tools/data_renewal_request.py

```diff
@@ -26,7 +26,6 @@
 total_count = 0
 updated_version = int(version) + 13
 for row in csv_reader:
-    (section, metric) = row["name"].split('.')
```

Comment:
Ran into an issue (in Focus) that Glean allow `.` in naming

Reply:
That's too bad. This doesn't look necessary with this iteration now anyway, but it was helpful for updating `metrics.yaml` with the results when I was trying to get that functionality to work as well.